Dockerfile

```diff
@@ -2,8 +2,15 @@ FROM alpine
 
 RUN apk add --no-cache bash tzdata
 
+RUN mkdir /config && \
+    mkdir /certs && \
+    mkdir /db && \
+    mkdir /decisions
+VOLUME ["/config", "/certs", "/db", "/decisions"]
```

Comment:
This change also now requires that all volumes are defined, which is not what we want in my  opinion, I want to be able to run with certs on the container image, I do not want to have to create a decisions volume if I am not using decision logs.

This is what is happening when using the current proposed Dockerfile
```
❯ docker run --name topaz-test-1 --rm --platform=linux/amd64 -e TOPAZ_DIR="/topaz_dir" -e TENANT_ID=$TENANT_ID -e DIRECTORY_SVC=$DIRECTORY_SVC -e DIRECTORY_KEY=$DIRECTORY_KEY -p 8282:8282 -p 8383:8383 -p 8484:8484 -p 8686:8686 -v $PWD:/config ghcr.io/aserto-dev/topaz:v0.30.28-amd64
{"level":"info","component":"config","time":"2024-01-20T18:46:41Z","message":"setting up certs for writer"}
{"level":"info","component":"cert-generator","common-name":"writer-grpc","cert-path":"/topaz_dir/certs/grpc.crt","key-path":"/topaz_dir/certs/grpc.key","ca-cert-path":"/topaz_dir/certs/grpc-ca.crt","time":"2024-01-20T18:46:41Z","message":"generating certificate"}
{"level":"info","log-source":"std","time":"2024-01-20T18:46:41Z","message":"2024/01/20 18:46:41 failed to setup certs: failed to generate grpc certs: directory verification returned an error: output directory '/topaz_dir/certs' doesn't exist\n"}
```